src/test/java/org/folio/rest/support/builders/RequestRequestBuilder.java

```diff
@@ -133,6 +141,14 @@ public JsonObject create() {
       put(item, "title", this.itemSummary.title);
       put(item, "barcode", this.itemSummary.barcode);
 
+      final JsonArray identifiers = new JsonArray(this.itemSummary.identifiers
```

Comment:
What happens if itemSummary.identifiers is null?

Reply:
It should not be null. It is initialized by an empty array and there is no setter for it (there is only an `addIdentifier` method).